eslint.config.js

```diff
@@ -64,7 +64,6 @@ export default defineConfig([
       '@crowdstrike/glide-core/no-nested-template-literals': 'error',
       '@crowdstrike/glide-core/no-redundant-property-attribute': 'error',
       '@crowdstrike/glide-core/no-redundant-property-string-type': 'error',
-      '@crowdstrike/glide-core/no-space-press': 'error',
```

Comment:
Figured we could remove these rules now given we won't be writing any new Web Test Runner tests.